website/docs/r/autoscaling_group.html.markdown

```diff
@@ -174,6 +174,51 @@ resource "aws_autoscaling_group" "example" {
 }
 ```
 
+### Mixed Instances Policy with Instance level LaunchTemplateSpecification Overrides
+
+When using a diverse instance set, some instance types might require a launch template with configuration values unique to that instance type such as a different AMI (Graviton), different EBS configuration.
```

Comment:
Slight wording tweak, since we're only listing two options
```suggestion
When using a diverse instance set, some instance types might require a launch template with configuration values unique to that instance type such as a different AMI (Graviton) or a different EBS configuration.
```

Reply:
I have updated the docs to include language more aligned with AWS's documentation on launch template overrides in a mixed instance policy. I specifically added the addition of architecture specific user data scripts, EBS configuration, and networking configuration. These features are especially important to customers running HPC workloads and large fleets with an instance type diversity strategy.